prime-router/src/main/kotlin/fhirengine/engine/FHIRTranslator.kt

```diff
@@ -68,7 +72,14 @@ class FHIRTranslator(
                 val receiver = settings.findReceiver(message.receiverFullName)
                     ?: throw RuntimeException("Receiver with name ${message.receiverFullName} was not found")
                 actionHistory.trackActionReceiverInfo(receiver.organizationName, receiver.name)
-                val bodyBytes = getByteArrayFromBundle(receiver, bundle)
+
+                val originalMessage = getOriginalMessage(message.reportId, WorkflowEngine())
+
+                val bodyBytes = if (originalMessage.schemaTopic.isSendOriginal) {
```

Comment:
Also since we're touching this function, can we update the else below to:

```
throw RuntimeException(
                    "Message was not a FhirTranslateQueueMessage and cannot be processed by FHIRTranslator: $message"
            )
```